### PR TITLE
Sprint 4

### PR DESCRIPTION
**Pull request by rowebyrowe**

### CODE REVIEW

**Review by veritaem — COMMENTED** [review]
this looks promising but it has no scores, Im definitely interested in seeing how good you did!

**Review by veritaem — COMMENTED** [review]
A little mean on the analysis but not terribly innaccurate :D it may also be a small company, one that doesnt have the resources to compete, so it naturally attracts people looking for a small town, cheaper alternative? eh

**Review by veritaem — COMMENTED** [review]
Your pdp plots and things look very nice but If I may quibble with a couple things, specifically your 'model.fit' and your train test split?

*   because the data in the df is ordered by date, and you just split it without shuffling, you lose a great deal of time series info (are more ppl buying bikes? Is pollution going up?). train_test_split has a default shuffle, it may help?
*   because you are estimating the best scoring model on your training data, the best score is just the one that fits all of your already labeled points (which is why its max depth/max trees), which will lead to some overfitting and a lesser prediction score. Have you tried making the model with a train score and predicting the val score to get your best model out that way?
